grpcutil/naming/naming.go

```diff
@@ -0,0 +1,43 @@
+package naming
```

Comment:
We don't use this code in grpc context. Should we move it into top-level naming package instead?

Reply:
@pstibrany done.